nixos/tests/miniflux.nix

```diff
@@ -81,7 +81,7 @@ in
           '';
         };
         systemd.services.postgresql-setup.postStart = lib.mkAfter ''
-          psql -tAd miniflux -c 'CREATE EXTENSION hstore;'
+          psql -tAd miniflux -c 'DROP EXTENSION IF EXISTS hstore;'
         '';
```

Comment:
This VM test has no state. The `IF EXISTS hstore` will never be `true`, because you replaced the `CREATE EXTENSION hstore` with it.

You can just drop this altogether:

```suggestion
```